docs/dev/tutorials/api3-usd-paymaster-tutorial.md

```diff
@@ -1,3 +1,10 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: USDC Paymaster tutorial | zkSync Era Docs
```

Comment:
nit/optional: `tutorial` is not capitalized here, but other titles we have it capitalized